runner/src/main.rs

```diff
@@ -50,15 +51,23 @@ use check_license::CheckLicense;
 mod check_build_licenses;
 use check_build_licenses::CheckBuildLicenses;
 
-#[cfg(target_os = "macos")]
-const DEFAULT_SERVER_RUST_TARGET: &str = "x86_64-apple-darwin";
-#[cfg(not(target_os = "macos"))]
-const DEFAULT_SERVER_RUST_TARGET: &str = "x86_64-unknown-linux-musl";
+use lazy_static::lazy_static;
 
 #[cfg(target_os = "macos")]
-const DEFAULT_EXAMPLE_BACKEND_RUST_TARGET: &str = "x86_64-apple-darwin";
+lazy_static! {
```

Comment:
I'll have a look at once_cell: Thanks for the pointer. I used the runtime composition here to avoid having to embed the multiple combinations in constants. Its not elegant, but it is hopefully more of a 'write once' approach.

Reply:
I knew there was a reasonI jumped for lazy_static - it supports a no_std spinlock implementation as well: If Oak never plans to be no_std (?), then I'll convert to once_cell.